src/app/avatar/api/heygen/newSession/route.tsx

```diff
@@ -0,0 +1,30 @@
+export async function POST(req: Request): Promise<Response> {
+  try {
+    console.log('POST request received');
+    const body = await req.json();
+    const { quality, avatar_name, voice_id } = body;
+    
+    const SERVER_URL = process.env.HEYGEN_SERVER_URL || '';
```

Comment:
maybe add some sort of doPost helper that minimizes the amount of boilerplate here

Reply:
ah yes! much cleaner now